tests/e2e/helpers.ts

```diff
@@ -63,3 +63,5 @@ export const isBeta = (): boolean => {
 
   return argparse.getArgValue('configuration').includes('beta');
 };
+
+export const delay = (ms: number) => new Promise(res => setTimeout(res, ms));
```

Comment:
root/utils/time.ts function `wait` does exactly that

Reply:
Thanks, I wasn't aware. However, I'm not convinced if we should mix a production and test code together. I need to think about it. 

For now, I'll simply remove it because I don't use this function anywhere - I've changed the conception in the meantime 😀